e2e/libs/node/node.py

```
@@ -34,14 +34,16 @@ def mount_disk(self, disk_name, node_name):
+        logging(f"Updating node {node_name} disks {disks}")
         for _ in range(self.retry_count):
             try:
                 node.diskUpdate(disks=disks)
                 self.wait_for_disk_update(node_name, len(disks))
-                break
+                return
             except Exception as e:
-                logging(f"Updating node {node_name} disk error: {e}")
+                logging(f"Failed to update node {node_name} disk: {e}")
             time.sleep(self.retry_interval)
+        assert False, f"Failed to update node {node_name} disk {disks}"
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)